release/release-debian

```diff
@@ -99,11 +99,11 @@ EOF
         reprepro -b $DEBIAN_REPO includedsc unstable cockpit_$debversion.dsc
 
         # build binary package
-        sudo pbuilder clean
-        sudo pbuilder update --configfile pbuilderrc --distribution unstable
-        sudo pbuilder build --configfile pbuilderrc \
-                            --hookdir /etc/pbuilder/hooks \
-                            cockpit_$debversion.dsc
+        basetgz=/var/cache/pbuilder/unstable.tgz
```

Comment:
Ok, that makes sense - this pull request was half-way ready for a loop, which made it appear strange. Improving it in a follow-up is acceptable.